miraclelinux/meta-emlinux
# Rename linux-k510 recipe file

# Purpose of pull request

Basically recipe file name has a suffix which describes software version
or source repositoy(e.g. git, debian). Adding a suffix to help poky
understands where source code come from.

before:
linux-k510:
  meta-emlinux         unknown (skipped)

after:
linux-k510:
  meta-emlinux         git (skipped)

# Test
## How to test

* Run bitbake-layer show-recipes to check linux-k510's version
* Build core-image-minimal


## Test result

DISTRO=emlinux

```
$ bitbake-layers show-recipes | grep -A 1 linux-k510 
linux-k510:
  meta-emlinux         git (skipped)
```

DISTRO="emlinux-k510"

```
$ bitbake-layers show-recipes | grep -A 1 linux-k510 
linux-k510:
  meta-emlinux         gitAUTOINC+3ddbe9bf6a
```

Build emlinux-k510 core-image-minimal

```
masami@ubuntu1804:~/emlinux/qemuarm64-build$ bitbake -f core-image-minimal                                                                                                                                                                    
Loading cache: 100% |##########################################################################################################################################################################################################| Time: 0:00:00
Loaded 2334 entries from dependency cache.                                                                                                                                                                                                    
NOTE: Resolving any missing task queue dependencies                                                                                                                                                                                           
                                                                                                                                                                                                                                              
Build Configuration:                                                                                                                                                                                                                          
BB_VERSION           = "1.42.0"                                                                                                                                                                                                               
BUILD_SYS            = "x86_64-linux"                                                                                                                                                                                                         
NATIVELSBSTRING      = "ubuntu-18.04"                                                                                                                                                                                                         
TARGET_SYS           = "aarch64-emlinux-linux"                                                                                                                                                                                                
MACHINE              = "qemuarm64"                                                                                                                                                                                                            
DISTRO               = "emlinux-k510"                                                                                                                                                                                                         
DISTRO_VERSION       = "2.3"                                                                                                                                                                                                                  
TUNE_FEATURES        = "aarch64 armv8a crc"                                                                                                                                                                                                   
TARGET_FPU           = ""                                                                                                                                                                                                                     
meta                                                                                                                                                                                                                                          
meta-yocto-bsp       = "warrior:c8c383d958807b991e31d22f612ba2a81a3860a4"                                                                                                                                                                     
meta-debian          = "warrior:2caf6c5cc97982ca335f4693cc8ccfedb71bf81c"                                                                                                                                                                     
meta-debian-extended = "warrior:bda787a691f29e4fa3b0084db2865a8b029b33c7"                                                                                                                                                                     
meta-emlinux         = "fix-linux-k510-recipe_name:879bfe30f206808880a5973c868d30f44be98ca7"                                                                                                                                                  
                                                                                                                                                                                                                                              
NOTE: Tainting hash to force rebuild of task /home/masami/emlinux/qemuarm64-build/../repos/poky/meta/recipes-core/images/core-image-minimal.bb, do_build                                                                       | ETA:  0:00:00
WARNING: /home/masami/emlinux/qemuarm64-build/../repos/poky/meta/recipes-core/images/core-image-minimal.bb.do_build is tainted from a forced run                                                                               | ETA:  0:00:00
Initialising tasks: 100% |#####################################################################################################################################################################################################| Time: 0:00:01
Sstate summary: Wanted 856 Found 853 Missed 3 Current 0 (99% match, 0% complete)                                                                                                                                                              
NOTE: Executing SetScene Tasks                                                                                                                                                                                                                
NOTE: Executing RunQueue Tasks                                                                                                                                                                                                                
NOTE: Tasks Summary: Attempted 3026 tasks of which 2795 didn't need to be rerun and all succeeded.                                                                                                                                            
                                                                                                                                                                                                                                              
Summary: There was 1 WARNING message shown.                  
```
